app/pages/index.tsx

```diff
@@ -1,129 +1,92 @@
+import { useState } from "react";
 import { GetServerSideProps } from "next";
-import cx from "classnames";
-import { CheckIcon } from "@heroicons/react/solid";
+import Head from "next/head";
 
 import { Progress } from "../components/Progress";
 import { ExternalLink } from "../components/Link";
+import { DosesDescriptionList } from "../components/DosesDescriptionList";
 import { DarkModeToggle } from "../components/DarkModeToggle";
-import { DhbPopulationDoseData } from "../types/VaccineDataTypes";
+import {
+  DhbRegionId,
+  DhbPopulationDoseDataWithRegion,
+} from "../types/VaccineDataTypes";
 import fetchHomePageProps from "../services/homePagePropsService";
 import { useHasMounted } from "../hooks/useIsMounted";
+import { RegionDropdown } from "../components/RegionDropdown";
 import dhbDisplayName from "../utilities/dhbDisplayName";
-import numberFormatter from "../utilities/numberFormatter";
 
 export type HomePageProps = {
-  allDhbsPopulationDoseData: DhbPopulationDoseData[];
-  aucklandDhbsPopulationDoseData: DhbPopulationDoseData[];
-  combinedAucklandDhbsPopulationDoseData: DhbPopulationDoseData;
+  allDhbsPopulationDoseData: DhbPopulationDoseDataWithRegion[];
   dataValidAsAtTimeUtc: string;
   dataFetchedAtTimeUtc: string;
 };
 
 const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
-  const {
-    combinedAucklandDhbsPopulationDoseData,
-    aucklandDhbsPopulationDoseData,
-  } = props;
-
-  const sortedAucklandDhbsPopulationDoseData =
-    aucklandDhbsPopulationDoseData.sort((a, b) =>
-      a.dhbName.localeCompare(b.dhbName)
-    );
-
+  const { allDhbsPopulationDoseData } = props;
   const hasMounted = useHasMounted();
+  const [region, selectedRegion] = useState<DhbRegionId>("auckland");
 
   return (
-    <div className="h-full min-h-screen">
-      <section className="pb-12 pt-12 md:pt-16">
-        <div className="max-w-[600px] mx-5 sm:mx-auto">
-          <div className="flex flex-row justify-content-between">
-            <div>
-              <h2 className="text-2xl font-light text-[color:var(--text-primary)]">
-                Vaccination rates
-              </h2>
-              <h1 className="text-6xl font-bold leading-[4.5rem] text-[color:var(--text-primary)] mb-6 md:mb-8">
-                Auckland
-              </h1>
+    <main className="page flex align-items-center">
+      <Head>
```

Comment:
We need to remove this `<Head>` and children too

Reply:
Ofc will wait for your PR too

Reply:
Should we merge that into this and have single release PR

Reply:
Na I think maybe even close this, there's two PRs now 😬

Reply:
I don't get it 😅 Can always merge around the same time